parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -101,18 +106,20 @@ installContainerdWithManifestJson() {
 }
 
 installContainerRuntime() {
+    local isKata=${1:-false}
```

Comment:
I figured `IS_KATA` would already be accessible from this file due to how it's `source`d, would rather just use that directly than add a parameter

Reply:
Oh that sounds good. Now I don't need it to be passed in. Thanks